metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/OrchestratorService.java

```diff
@@ -288,14 +290,14 @@ public WorkflowExecution getWorkflowExecutionByExecutionId(MetisUser metisUser,
    * </ul>
    */
   public WorkflowExecution addWorkflowInQueueOfWorkflowExecutionsWithoutAuthorization(
-      String datasetId, ExecutablePluginType enforcedPredecessorType, int priority)
+      String datasetId, Workflow workflow, ExecutablePluginType enforcedPredecessorType, int priority)
```

Comment:
Rename to match the name in the method below (`workflowProvided`)? Or perhaps something else (e.g. `alternativeBaseWorkflow`)?

Reply:
ok